gulpfile.js/tasks/drizzle.js

```diff
@@ -3,9 +3,30 @@
 const drizzle = require('drizzle-builder');
 const config = require('../config');
 const helpers = require('@cloudfour/hbs-helpers');
+const Handlebars = require('handlebars');
+
+// Register handlebars helpers
+function renderIcons(iconData, depth=2){
+    let output = '';
+    Object.keys(iconData)
+        .forEach( (key) => {
+            output += `<h${depth} class="icon-title">${key}</h${depth}>`;
+            iconData[key].files.forEach( (file) => {
+                output += `<div class="img-group" data-clipboard-text="img/icons/${file.src}"><img src="/assets/protocol/protocol/img/icons/${file.src}" class="icons"/>`;
```

Comment:
We can add `title="Click to copy file path"` to offer a hint, along with the cursor change.